### ISSUE TITLE
Change the error text styling

### ISSUE THREAD

**Issue by brandonkelly** [craftcms/cms#2885]
There is a form of color blindness where our red error text color is indistinguishable from normal text, making it impossible to spot the errors on a page.

**Comment by lukeyouell** [comment]
@brandonkelly on the topic of accessibility, having white text over the orange background in the error banner fails the [colour contrast check](https://webaim.org/resources/contrastchecker/):

<img width="746" alt="image" src="https://user-images.githubusercontent.com/24975058/40105591-4b0caf3e-58eb-11e8-9697-479d25c76f45.png">

**Comment by brandonkelly** [comment]
@lukeyouell Done :)
